server/server.go

```diff
@@ -634,6 +636,19 @@ func (s *Server) v1DiagnosticsGet(w http.ResponseWriter, r *http.Request) {
 	writer.JSON(w, 200, resp, pretty)
 }
 
+func (s *Server) unversionedGetHealth(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	// Create very simple query that binds a single variable.
+	rego := rego.New(rego.Query("x = 1"))
```

Comment:
Pass the server's compiler & store when when constructing the rego object. This'll help ensure that we're testing the same code paths that are used during normal evaluation.

Reply:
Done